examples/mobile/UIComponents/css/style.css

```diff
@@ -720,4 +720,8 @@
 	left: 7px;
 	font-size: 12px;
 	color: white;
+}
+
+.app-full-height {
+	height: 100vh;;
```

Comment:
Would you remove the duplicated ";" please?

Reply:
Is it correct the "vh" unit?

Reply:
Fixed.
This is proper unit. The "vh" means percent of view. For the require of the example, I need to extend an empty list. Well, this is not perfect way but it works almost as intended